config/README.md

```diff
@@ -75,8 +75,10 @@ Storage backends:
 - [Ceph](storage/ceph.md)
 - [Redis](storage/redis.md)
 - [OpenStack Swift](storage/swift.md)
-- [Loclahost](storage/localhost.md)
```

Comment:
@JosepSampe we don't need localhost anymore?

Reply:
ops! deleted by error